lib/modules/manager/types.ts

```diff
@@ -33,6 +33,7 @@ export interface CustomExtractConfig
 export interface UpdateArtifactsConfig {
   isLockFileMaintenance?: boolean;
   constraints?: Record<string, string>;
+  conservative?: boolean;
```

Comment:
```suggestion
```

I think this can be removed?

Reply:
Whoops; thought I already had; thanks for catching that.